fenix/app/src/main/java/org/mozilla/fenix/settings/search/SearchShortcutsFragment.kt

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.search
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import kotlinx.coroutines.MainScope
+import mozilla.components.browser.state.search.SearchEngine
+import mozilla.components.support.ktx.android.view.hideKeyboard
+import org.mozilla.fenix.R
+import org.mozilla.fenix.databinding.FragmentSearchShortcutsBinding
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.getRootView
+import org.mozilla.fenix.ext.requireComponents
+import org.mozilla.fenix.ext.settings
+import org.mozilla.fenix.ext.showToolbar
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+import org.mozilla.fenix.utils.allowUndo
+
+/**
+ * A [Fragment] that allows user to select what search engine shortcuts will be visible in the quick
+ * search menu.
+ */
+class SearchShortcutsFragment : Fragment(R.layout.fragment_search_shortcuts) {
+
+    lateinit var tabsTrayDialogBinding: FragmentSearchShortcutsBinding
```

Comment:
I believe this will want to be nullable and handled by `onDestroyView` as is done with the binding in `SaveSearchEngineFragment`.

Reply:
I also see the name is `tabsTrayDialogBinding` so that should be updated